core/sequencer/server.go

```diff
@@ -418,6 +419,7 @@ func (s *Server) ApplyRevision(ctx context.Context, in *spb.ApplyRevisionRequest
 
 	// Marshal new indexed values back into Trillian Map leaves.
 	newLeaves := runner.DoMarshalIndexedValues(newIndexedLeaves, emitErrFn, incMetricFn)
+	fnLatency.Observe(time.Since(computeStart).Seconds(), in.DirectoryId, "Compute")
```

Comment:
Maybe there is a less generic name for "Compute"? E.g. "(Apply|Compute|Process)Mutations"?